<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>571</width>
       <height>481</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="selectImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_resize">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_horizontalflip">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_randomsizecrop">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_centercrop">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_rgbshift">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_blur">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_vertflip">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>200</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_clahe">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>180</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_randombrightnesscontrast">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>160</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_elastictransform">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>220</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_griddistortion">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>420</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_multiplicativenoise">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>300</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_imagecompression">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>320</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_togray">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>440</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_equalize">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>400</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_tosepia">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>260</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_multi">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>280</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_colorjitter">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>360</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_emboss">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>380</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_randomcontrast90">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>240</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_channeldropout">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>340</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_resize">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>40</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_horizontalflip">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>60</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_centercrop">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>100</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_randomsizecrop">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>80</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_blur">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>140</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_rgbshift">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>120</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_randombrightnesscontrast">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>160</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_vertflip">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>200</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_clahe">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>180</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_randomcontrast90">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_togray">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>440</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_channeldropout">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>340</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_tosepia">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>260</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_emboss">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>380</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_equalize">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>400</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_multiplicativenoise">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>300</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_elastictransform">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>220</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_griddistortion">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>420</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_colorjitter">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>360</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_multi">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>280</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_imagecompression">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>320</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="todo0">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="todo1">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="allg">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="imageFolder">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="labelFolder">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Label Folder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>resize</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>horizontalflip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>randomsizecrop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>centercrop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>140</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>blur    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>rgbshift</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>160</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>randombrightnesscontrast</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>180</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>clahe    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>vertflip    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>tosepia    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>280</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>multi    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>240</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>randomcontrast90    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>220</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>elastictransform    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>620</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resize</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>340</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>channeldropout    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>660</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resize</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>300</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>multiplicativenoise    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>360</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>colorjitter    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>320</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>imagecompression      </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>420</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>griddistortion    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>640</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resize</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>440</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>togray    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>400</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>equalize    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>380</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>emboss    </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>460</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>opt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>480</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>vertflip    </string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="opt">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>480</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="opt2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>460</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="opt4">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>480</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>460</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>521</y>
       <width>971</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="gone">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>G_ONE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gall">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>G_All</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="labelcheckbox">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
